e2e/testdata/fn-render/basicpipeline/Kptfile

```diff
@@ -4,9 +4,9 @@ metadata:
   name: app
 pipeline:
   mutators:
-    - image: gcr.io/kpt-fn/set-namespace:v0.1.3
+    - image: set-namespace:v0.1.3
```

Comment:
Instead of editing an existing test case, add a new ones. This is purely an additive change which should not touch existing test coverage.

Reply:
Done.